maven-plugins/run-automation-package-sample/pom.xml

```diff
@@ -0,0 +1,223 @@
+                                <phase>package</phase>
+                                <configuration>
+                                    <url>${step-server-url}</url>
+
+                                    <artifactClassifier>tests</artifactClassifier>
+                                    <description>My test execution</description>
+
+                                    <waitForExecution>true</waitForExecution>
+                                    <ensureExecutionSuccess>true</ensureExecutionSuccess>
+
+                                    <!-- Custom execution parameters -->
+                                    <executionParameters>
+                                        <env>PROD</env>
+                                        <PARAM_EXEC>Value</PARAM_EXEC>
+                                        <PARAM_EXEC2>Value</PARAM_EXEC2>
+                                        <PARAM_EXEC3>Value</PARAM_EXEC3>
+                                    </executionParameters>
+                                </configuration>
+                                <goals>
+                                    <goal>run-packaged-automation-packages-os</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <!-- Configuration for EE version (maven profile) -->
+        <profile>
+            <id>StepEE</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>ch.exense.step</groupId>
+                            <artifactId>step-ee-maven-plugin</artifactId>
+                            <version>${step-maven-plugin.version}</version>
+                        </plugin>
+                    </plugins>
+                </pluginManagement>
+
+                <plugins>
+                    <plugin>
+                        <groupId>ch.exense.step</groupId>
+                        <artifactId>step-ee-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>deploy-packaged-to-step</id>
+                                <phase>package</phase>
+                                <configuration>
+                                    <url>${step-server-url}</url>
+
+                                    <artifactClassifier>tests</artifactClassifier>
+                                    <description>My test execution</description>
+
+                                    <waitForExecution>true</waitForExecution>
+                                    <ensureExecutionSuccess>true</ensureExecutionSuccess>
+
+                                    <authToken>
```

Comment:
Would use a variable here and declare it after the step url with a comment stating that it is only required for Step EE